### PR TITLE
Disable path rewrite for default locale

### LINKED ISSUE #6: Add a feature to $tp, so default language is not prefixed
Thank you for your working. This package helps a lot! I would love to add one feature to `$tp`. 

With my current settings I generate the links like this:

```
/                    -> component home
/it-it/              -> component home
/en-gb/              -> component home
/en-gb/about         -> component about
/it-it/about         -> component about
/about               -> component about
/it-it/about         -> component about
```

For each link without prefix, the default language is being used. So now using $tp it forces me to `en-gb`. And I don't want to add a prefix only if it's a foreign language (looked at the domain ending).

For more e.g. https://github.com/daaru00/gridsome-plugin-i18n/pull/2#issuecomment-612656753

I hope it's understandable

### PR DESCRIPTION
**Pull request by daaru00**

### CODE REVIEW

File: gridsome.client.js [hunk 1]
```diff
@@ -77,26 +89,22 @@ export default function (Vue, options, { appOptions, router }) {
   router.beforeResolve((to, from, next) => {
     // do not rewrite build paths
     if (process.isServer) {
-      next()
-      return 
+      return next()
     }
 
     // if option is disabled skip whole logic
     if (options.enablePathRewrite === false) {
```
Comment (cfecherolle) [inline gridsome.client.js:101]: I tried changing this option and setting `enablePathRewrite: false` in my site settings but paths were still changed and prefixed when switching locale in my page (by clicking a flag, for example). Maybe I didn't understand the behaviour, I'm not sure

**Review by cfecherolle — COMMENTED** [review]
I reviewed the code and added a remark, apart from that I tested the new option and it works fine!

File: gridsome.client.js (hunk 1 above)
Comment (daaru00) [reply gridsome.client.js:101]: That option will skip a route redirect used to keep the translated parts in path when you don't use the `$tp` for all links in page. Is not strictly related to language switch (are you switch locale from `$i18n` or just using a path redirect?) but with other liinks.

For example you have an homepage with these links:
```html
<g-link to="/blog"> Blog </a>
<g-link to="/who-we-are"> Who we are </a>
```
and you're not using `$tp` to translate it. When you click on the link the `beforeResolve` it checks the current locale and perform a redirect if `/en/`, `/de/`  if not present.

When you switch of `enablePathRewrite` and you click on these links the path remain untouched and basically use the fallback locale. 

Not sure I healed your doubt :sweat_smile:

Comment (cfecherolle) [reply gridsome.client.js:101]: Ok I get it now! Thank you for the explanation :)

**Comment by daaru00** [comment]
Just rebased with latest master changes to avoid some conflicts

**Comment by daaru00** [comment]
It seems does not break nothing until now (used on a live website pointing the feature branch). 

@cfecherolle if you have nothing against it, I'll would proceed with merge

**Comment by cfecherolle** [comment]
If you tested it, sure, I'm okay with that! Go on and merge :)

**Review by cfecherolle — APPROVED** [review]

**Pull request merged**